cluster-sync/ephemeral_provider.sh

```diff
@@ -49,10 +49,14 @@ function configure_storage() {
 }
 
 function configure_hpp() {
-  for i in $(seq 1 ${KUBEVIRT_NUM_NODES}); do
-      ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo mkdir -p /var/hpvolumes"
-      ./cluster-up/ssh.sh "node$(printf "%02d" ${i})" "sudo chcon -t container_file_t -R /var/hpvolumes"
-  done
+  if [[ $KUBEVIRT_PROVIDER =~ kind.* ]]; then
+      ./cluster-up/ssh.sh kind-1.28-control-plane mkdir -p /var/hpvolumes
```

Comment:
is there maybe a more generalized way to get the node's name? so we don't have to keep the k8s version string in sync here